composer.json

```diff
@@ -29,10 +29,10 @@
     ],
     "homepage": "https://www.doctrine-project.org",
     "require": {
-        "php": "^7.1 || ^8.0",
+        "php": "^7.3 || ^8.0",
```

Comment:
PHP 7.3 is supported by unmaintained DBAL 3 branches only. We should bump to PHP 7.4 directly.

See also the [statistics on Packagist](https://packagist.org/packages/doctrine/dbal/php-stats#3): PHP 7.3 accounts for only 1.8% of all DBAL 3 installations.